src/mr_allocator.rs

```diff
@@ -217,6 +329,29 @@ fn get_default_hooks_impl(arena_ind: u32) -> extent_hooks_t {
     unsafe { *hooks }
 }
 
+/// Alloc raw memory region for RDMA operations
+///
+/// # Safety
+///
+/// This func is safe when:
+/// * `flag == MALLOCX_ZERO`, otherwise newly allocated memory is uninitialized.
+/// * Dealloc `addr` after dropping `raw_mr`.
+unsafe fn alloc_raw_mem(layout: &Layout, flag: i32) -> Option<*mut u8> {
+    let addr = if flag == MALLOCX_ZERO {
+        // // SAFETY: dst is valid for writes of `layout.size() * size_of<u8>` bytes and has been properly aligned.
```

Comment:
It's not the unsafe bound, why do we add SAFETY notation here?

Reply:
This is an omission, I will correct it.